dist/api_data.py

```diff
@@ -166,7 +166,9 @@ def __ge__(self, other):
             Config('max_consecutive_delta', '32', r'''
                 the max consecutive deltas allowed for a single page. The maximum value is set
                 at 32 (WT_DELTA_LIMIT). If we need to change that, please change WT_DELTA_LIMIT
-                as well.''', min='1', max='32')
+                as well.''', min='1', max='32'),
+            Config('max_delta_size', '6', r'''
```

Comment:
No, I think it should be full_page_threshold. If the full page size is smaller than this value, we always write a full page.